front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -208,6 +221,51 @@ export const usePathfinding = (
     }
   }, [origin?.id, destination?.id, rollingStock]);
 
+  const handleInvalidPathItems = (
+    pathfindingResult: PathfindingResult,
+    pathSteps: (PathStep | null)[],
+    dispatch: AppDispatch,
+    pathfindingDispatch: React.Dispatch<PathfindingAction>,
+    t: TFunction
+  ) => {
+    if (
+      pathfindingResult.status === 'failure' &&
+      pathfindingResult.failed_status === 'pathfinding_input_error' &&
+      pathfindingResult.error_type === 'invalid_path_items'
+    ) {
```

Comment:
Since you have the invalidItems in the arguments of your function, you can remove this if
```suggestion
```

Reply:
done